petstore/frontend/react/src/App.test.js

```diff
@@ -1,9 +0,0 @@
-import React from 'react';
```

Comment:
I deleted this file because the test was failing due to use of EventSource not being supported by jsdom. We could install a polyfill or mock version of that, but this test wasn't really doing much currently.